gradle.properties

```diff
@@ -3,7 +3,7 @@
 
 org.gradle.parallel=true
 
-org.gradle.jvmargs=-Xmx4g -Xss4m
+org.gradle.jvmargs=-Xmx8g -Xss4m
```

Comment:
This was needed to make gradle to be able to import the new module

Reply:
Very strange.  It seems odd that just moving the code to a library would cause Gradle to need more memory.  We should probably look into this a bit, because needing 8 GB to build the server seems like that may be an issue for OSS users.

Reply:
I agree, I ran into java heap memory issue. I cleaned the cache which didn't work then re-clone the project which ran into the same issue, it was the only way I found to include the new module. I haven't tried with a lower memory setting.